src/instance.ts

```diff
@@ -26,9 +28,32 @@ export class ExtensionInstance {
 	constructor(context: ExtensionContext) {
 		this.context = context;
 
-		this.networkComponent = new NetworkComponent(this);
+		const controlInstance = spawn("node", [ 
```

Comment:
This also shouldn't be a constant, but added to the class as a public field, exposure of this process would help with flexibility at a later date

Reply:
-1